draft-ietf-quic-version-negotiation.md

```diff
@@ -65,6 +65,10 @@ receives one. QUIC version 1 {{!QUIC=RFC9000}} allows the server to use a
 Version Negotiation packet to indicate that the version the client chose is
 unacceptable, but doesn't allow the client to safely make use of that
 information to create a new connection with a mutually supported version.
+This document updates {{QUIC-INVARIANTS}} by defining an alternative version
+negotiation mechanism, while keeping the option to use the mechanism defined in
+{{QUIC-INVARIANTS}} for negotiating versions incompatible with the initially
```

Comment:
This text contradicts the start of the paragraph, QUIC-INVARIANTS does not define a version negotiation *mechanism*, it only defines a version negotiation *packet*.